.github/workflows/pest-tests.yml

```
@@ -0,0 +1,91 @@
+name: Unit & Feature Tests
+
+on:
+  push:
+    branches: [ "master" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)